src/conventions/h-group/take-action.js

```diff
@@ -263,10 +259,22 @@ export function take_action(game) {
 		}
 	}
 
+	// Consider finesses while finessed if we are only waited on to play one card,
+	// it's not a selfish finesse, doesn't require more than one play from our own hand,
+	// and we're not in the end-game.
+	const waiting_self_connections = game.common.waiting_connections.filter(c => c.connections[0]?.reacting === state.ourPlayerIndex).length;
+	const is_selfish = best_play_clue?.result.playables[0]?.playerIndex === state.ourPlayerIndex;
+	const self_rounds = best_play_clue?.result.playables.filter(play => play.playerIndex === state.ourPlayerIndex).length;
```

Comment:
My idea for this condition was more for the finesse that we would be deferring, I should have been more specific. I'd say we should not defer if there are:
- 3+ cards to play after ours
- 2 cards to play after ours and the next player is not in between us and the third player (e.g. we are Alice with r2, Bob has r4, Cathy has r3, don't defer). This should be exclusive (i.e. we are Alice with r2 and r3, Bob has r4, don't defer).

Reply:
Thanks for the suggestion! I think I've got this fully implemented.